.github/workflows/test-documentation.yml

```diff
@@ -0,0 +1,127 @@
+        id: buildx
+        uses: docker/setup-buildx-action@v3
+  
+      - name: Create Build Image
+        uses: docker/build-push-action@v5
+        with:
+          builder: ${{ steps.buildx.outputs.name }}
+          file: ${{ github.workspace }}/HPCC-Platform/dockerfiles/vcpkg/${{ inputs.os }}.dockerfile
+          context: ${{ github.workspace }}/HPCC-Platform/dockerfiles/vcpkg
+          push: false
+          load: true
+          build-args: |
+            VCPKG_REF=${{ steps.vars.outputs.vcpkg_sha_short }}
+          tags: |
+            ${{ steps.vars.outputs.docker_tag_candidate_base }}
+          cache-from: |
+            type=registry,ref=${{ steps.vars.outputs.docker_tag_candidate_base }}
+            type=registry,ref=${{ steps.vars.outputs.docker_tag }}
+          cache-to: type=inline
+
+      - name: CMake documentation
+        run: |
+          mkdir -p {${{ github.workspace }}/build,EN_US,PT_BR}
+          docker run --rm --mount source="${{ github.workspace }}/HPCC-Platform",target=/hpcc-dev/HPCC-Platform,type=bind,consistency=cached --mount source="${{ github.workspace }}/build",target=/hpcc-dev/build,type=bind,consistency=cached ${{ steps.vars.outputs.docker_tag_candidate_base }} "\
+            cmake -S /hpcc-dev/HPCC-Platform -B /hpcc-dev/build -DVCPKG_FILES_DIR=/hpcc-dev -DMAKE_DOCS_ONLY=ON -DUSE_NATIVE_LIBRARIES=ON -DDOCS_AUTO=ON -DDOC_LANGS=ALL && \
+            cmake --build /hpcc-dev/build --parallel $(nproc) --target all"
+          docker run --rm --mount source="${{ github.workspace }}/HPCC-Platform",target=/hpcc-dev/HPCC-Platform,type=bind,consistency=cached --mount source="${{ github.workspace }}/build",target=/hpcc-dev/build,type=bind,consistency=cached ${{ steps.vars.outputs.docker_tag_candidate_base }} "cd /hpcc-dev/build/Release/docs/EN_US && zip ALL_HPCC_DOCS_EN_US-$(echo '${{ steps.vars.outputs.community_tag }}' | sed 's/community_//' ).zip *.pdf"
+          docker run --rm --mount source="${{ github.workspace }}/HPCC-Platform",target=/hpcc-dev/HPCC-Platform,type=bind,consistency=cached --mount source="${{ github.workspace }}/build",target=/hpcc-dev/build,type=bind,consistency=cached ${{ steps.vars.outputs.docker_tag_candidate_base }} "cd /hpcc-dev/build/Release/docs/PT_BR && zip ALL_HPCC_DOCS_PT_BR-$(echo '${{ steps.vars.outputs.community_tag }}' | sed 's/community_//' ).zip *.pdf"
+
+      - name: Upload build artifacts
+        uses: actions/upload-artifact@v4
+        with:
+          name: ${{ inputs.asset-name }}
+          path: |
+            ${{ github.workspace }}/build/Release/docs/*.zip
+            ${{ github.workspace }}/build/Release/docs/EN_US/*.zip
+            ${{ github.workspace }}/build/Release/docs/PT_BR/*.zip
+            ${{ github.workspace }}/build/docs/EN_US/EclipseHelp/*.zip
+            ${{ github.workspace }}/build/docs/EN_US/HTMLHelp/*.zip
+            ${{ github.workspace }}/build/docs/PT_BR/HTMLHelp/*.zip
+          compression-level: 0
```

Comment:
Are there any logs that could be uploaded on failure?

Reply:
Currently there aren't any logs for doc builds or failures, outside of what is piped to stderr/stdout.  I looked into adding them, but it requires me to do a lot of stuff that might break doc builds. I'd rather not include it here.